tests/Fixer/Whitespace/NoTrailingWhitespaceFixerTest.php

```diff
@@ -96,6 +96,37 @@ public function provideFixCases()
             [
                 "<?php\necho <<<'EOT'\nInline Il y eut un   \r\nrire éclatant    \n     \n   \r\nEOT;\n\n",
             ],
+            [
+                "<?php\necho 'Hello World';",
+                "<?php \necho 'Hello World';",
+            ],
+            [
+                "<?php\n\necho 'Hello World';",
+                "<?php \n\necho 'Hello World';",
+            ],
+            [
+                "<?php\r\necho 'Hello World';",
+                "<?php \r\necho 'Hello World';",
+            ],
+            [
+                "<?php\necho 'Hello World';",
+                "<?php  \necho 'Hello World';",
+            ],
+            [
+                "<?php\necho 'Hello World';",
+                "<?php	\necho 'Hello World';",
+            ],
+            [
+                '<?php ',
+                '<?php  ',
+            ],
+            [
+                "<?php\t",
+                "<?php\t\t",
+            ],
+            [
+                '<?php ', // do not trim this as "<?php" is not valid PHP
+            ],
```

Comment:
please add this case;
```php
[
"<?php\n      \n   \n    "',
]
```

Reply:
@SpacePossum added, I hope you mean 2nd one with fix (as the first set of spaces is in non-blank line).